### ISSUE TITLE
Update app icons

### ISSUE THREAD

**Issue by hellyab** [LeafGlobalFintech/loan_app#49]
Currently, both the iOS and Android versions of the app use the flutter logo as app icons. We want to replace them with the leaf logo that is attached to the image. 
To get the icons for different resolutions one can use [appicon.co](https://appicon.co/) or [easyappicon.com](https://easyappicon.com/)

Replacing drawables in `android\app\src\main\res` for android and replacing the contents of `ios\Runner\Assets.xcassets\AppIcon.appiconset` for iOS should do the trick.

Try not to get rid of other assets and also make sure the naming stays similar to the one already in the app to avoid having to change a lot of references as that makes reviewing a PR tougher.

![logo](https://user-images.githubusercontent.com/39404675/155559662-0ca34145-c3af-47f1-9a2b-ca4d4e75fc89.jpg)

**Comment by AE542** [comment]
Hi hellyab, I could replace the iOS AppIcon set with one using Leaf's logo. I don't know how to do it for android however. Is it ok to tackle the iOS side of it?

**Comment by hellyab** [comment]
Hey @AE542, sure thing. The android version shouldn't be that tough too. I could possibly send you an article if you are interested.

**Comment by AE542** [comment]
Yeah I actually had a look and it doesn't seem so bad. I'll try for both versions. Also sorry just an other question. To get the loan app to run on my computer, I'll need flutter, Firebase and credo labs auth key and endpoint url? Is that right?

**Comment by hellyab** [comment]
Yeah, you can create a firebase project and use it to build and you can use a dummy enpoint that accepts post for the credo labs api url with a dummy api key. [https://credo-dummy.free.beeceptor.com/my/api/path](https://credo-dummy.free.beeceptor.com/my/api/path) is set up to accept post requests so it should be fine.

**Comment by hellyab** [comment]
But in theory, you don't have to go through all those, you can just `flutter build apk` and see if the app has the icon (on Android) and run `flutter build ipa` for iOS.

**Comment by AE542** [comment]
> But in theory, you don't have to go through all those, you can just `flutter build apk` and see if the app has the icon (on Android) and run `flutter build ipa` for iOS.

Hi, I've managed to change iOS app icon assets while keeping the names of the new icons the same as well as all the correct sizes. However I still can't get the app to run because apparently there's 'no XCFramework for the credoappsdk.xcframework' even though there is. I tried running `flutter build ipa` but I get the same error. The icons should show no problem on iOS devices however. Could I at least pass on the updated iOS icons to you guys? I'm still working on the Android ones.
